lib/main.js

```diff
@@ -149,7 +149,7 @@ const Hydrogen = {
     store.subscriptions.add(
       atom.commands.add("atom-workspace", {
         "hydrogen:clear-results": () => {
-          store.markers.clear();
+          store.markersMapping.get(store.editor.id).clear();
```

Comment:
Is flow ok with this? What if editor is null? 💥

I think the solution is to hide the markermap detail I'm the store like we do kernelmapping. Do a null check there once in the getter, then `store.marker` is fine

Reply:
Flow didn't complain but yes I think your solution is definitely cleaner